content/english/phaser-space-invaders-game/activity-2.md

```diff
@@ -24,8 +28,10 @@ this.background.setOrigin(0, 0);
 
 TileSprites will allow our background to move even though it's not a spritesheet because it is a type of Sprite with repeating texture, so it can be scrolled through Phaser's infinite Canvas.
```

Comment:
```suggestion
`TileSprites` will allow our background to move even though it's not a spritesheet because it is a type of Sprite with repeating texture, so it can be scrolled through Phaser's infinite Canvas.
```